app/views/results.nunjucks

```diff
@@ -7,7 +7,7 @@
 {% endblock %}
 
 {% block content %}
-<a href="{{ SITE_ROOT }}/find-help?context={{ context }}" class="link-back">Back to find a GP</a>
+<a href="{{ SITE_ROOT }}?context={{ context }}" class="link-back">Back to find your GP</a>
```

Comment:
Should `GP` be `GP Surgery` given that would be the singular version of what is displayed on line 22?